modules/kubectl/init.zsh

```diff
@@ -0,0 +1,40 @@
+#
+# Provides 'kubectl' aliases and utiities.
+#
+# Authors:
+#   Bruno Miguel Custodio <brunomcustodio@gmail.com>
+#
+
+# Return if requirements are not found.
+if (( ! ${+commands[kubectl]} )); then
+  return 1
+fi
+
+# Enable completion for 'kubectl'.
+cache_file="${0:h}/cache.zsh"
+kubectl_command="${commands[kubectl]}"
+
+if [[ "${kubectl_command}" -nt "${cache_file}" || ! -s "${cache_file}" ]]; then
```

Comment:
What's the purpose of `"${kubectl_command}" -nt "${cache_file}"`? I saw this on node module as well, but couldn't recall why it is there. @belak any recollection?